### DISCUSSION TITLE
How to serve static file in deno?

### DISCUSSION THREAD

**Discussion by molystechie** [honojs/hono#2755] [Q&A]
hi am in difficult time to work with static files, i read the doc but still not get it correctly
This from doc how to serve static file: https://hono.dev/getting-started/deno#serve-static-files

app.use('/static/*', serveStatic({ root: './' }))
app.use('/favicon.ico', serveStatic({ path: './favicon.ico' }))

folder structure
./
├── favicon.ico
├── index.ts
└── static
    ├── demo
    │   └── index.html
    ├── fallback.txt
    ├── hello.txt
    └── images
        └── dinotocat.png

if in static directory i have js/script.ts, what would be its script in head?
favicon.svg dont show up, how to to put it correctly?

in HTML head 
        <head>
            <title>${title}</title>
            <link rel="icon" type="image/svg+xml" href="./favicon.svg" />
            <script type="text/javascript"  src="./js/script.ts" defer></script>
        </head>

**Comment by NicoPlyley** [comment]
I believe the static folder needs to be in your root directory, not inside the src folder. Let me know and we can get the docs updated accordingly.

**Reply by JulianMiguelAngle** [reply]
Yeah dude the docs is confussing
